config-helpers.go

```diff
@@ -34,9 +35,9 @@ func envConfigIntWithDefault(key string, defValue int) int {
 // Parsing the "bucket/path:project/key,bucket2:key2"
 func bucketKeyMappings(bucketKeyMapString string) map[string]string {
 
-	fmt.Println(bucketKeyMapString)
+	log.Debug(bucketKeyMapString)
```

Comment:
it's better to provide what's being logged. something like log.Debugf("keyMapString: %v", bucketKeyMapString)